torch/csrc/jit/passes/quantization/helper.cpp

```diff
@@ -668,5 +668,17 @@ bool is_batchnorm3d_module(
       "__torch__.torch.nn.modules.batchnorm.BatchNorm3d");
 }
 
+bool is_fp16_fp32_cast_op(
+    const Match& match,
+    const std::unordered_map<std::string, Value*>& vmap) {
+  const auto& match_vmap = match.values_map;
+  auto fp16_type = toIValue(match_vmap.at(vmap.at("dtype_fp16")));
+  auto fp32_type = toIValue(match_vmap.at(vmap.at("dtype_fp32")));
```

Comment:
also would be good to split these as well, so the filters can be more composable

Reply:
you can add a function similar to `is_int_constant` for scalar type